manticore/core/manticore.py

```diff
@@ -365,6 +351,8 @@ def __init__(self, initial_state, workspace_url=None, outputspace_url=None, **kw
         # Note that each callback will run in a worker process and that some
         # careful use of the shared context is needed.
         self.plugins = set()
+        self._introspection_plugin: type = IntrospectionAPIPlugin
+        self._introspector: typing.Optional[IntrospectionAPIPlugin] = None
```

Comment:
`_introspector` is the IntrospectionAPIPlugin instance. The only reason to separate it out is so that `ManticoreBase.introspect` can reference it [later](https://github.com/trailofbits/manticore/pull/1775/files#diff-29da5f65e4b187c1f150f160571ea968R1202)

Reply:
I tweaked the `ManticoreBase` constructor to add the `introspection_plugin_type` parameter, which eliminates the need for the `_introspection_plugin` field here (as well as the helper method that sets it)